llvm/test/MC/AArch64/local-bounds-single-trap.ll

```diff
@@ -0,0 +1,83 @@
+; RUN: llc -O3 -mtriple arm64-linux -filetype asm -o - %s | FileCheck %s -check-prefix CHECK-ASM
+; What this test does is check that even with nomerge, the functions still get merged in
+; compiled code as the ubsantrap call gets lowered to a single instruction: brk.
```

Comment:
you don't change MC, so I don't think we need the test here

However we need a test in llvm-project/llvm/test/Instrumentation/BoundsChecking/

Reply:
I understood your comment on the Phabricator diff as a request for this test:

> can you please create a test where bounds-checking-single-trap=0 and setCannotMerge produce invalid result.

I will also add a test in `llvm/test/Instrumentation/BoundsChecking/`